playwright-e2e/playwright.config.ts

```diff
@@ -81,70 +91,31 @@ const testConfig: PlaywrightTestConfig = {
       fullPage: true
     },
     video: 'retain-on-failure', // Limit load on CI system because trace and video add load
-
-    userAgent: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36',
-    viewport: { width: 1280, height: 960 },
-    ignoreHTTPSErrors: true
-    // launchOptions: {
-    //   ignoreDefaultArgs: ['--hide-scrollbars'],
-    // },
   },
 
   /* Configure projects for major browsers */
   projects: [
     {
-      name: 'chromium',
-      // testMatch: '**/*.spec.ts',
-      grepInvert: /examples/,
-      use: {
-        browserName: 'chromium',
-        launchOptions: {
-          slowMo: 100,
-          // Account for minor difference in text rendering and resolution between headless and headed mode
-          ignoreDefaultArgs: ['--hide-scrollbars']
-        }
-      }
+      // Listing tests: npx playwright test --list --project="dsm"
+      name: 'dsm',
```

Comment:
`--project` allows us to run set of tests. For example, to run all DSM tests, run cmd: `npx playwright test --project="dsm"`.

To run all tests, cmd still is `npx playwright test`.